---
layout: page
page_name: software
show_sidebar: false
title: Software
description: Software Description
toc: false
show_sidebar: false
hide_footer: false
---
# Software

#### Please report all software questions/issues here

[Please report all software questions/issues here](https://gitreports.com/issue/parklab/Software-Issues?email_public=on)


#### Note: Our software packages are free for academic use. For commercial use, please contact the Harvard Office of Technology Development (hms_otd@harvard.edu) with cc to Dr. Park. Patents have been filed for some algorithms.

[hms_otd@harvard.edu](mailto:hms_otd@harvard.edu)

## New!

SigMA(https://github.com/parklab/SigMA)For detection of homologous recombination deficiency (Signature 3) from panel, exome, or whole-genome data. See Gulhan et al, Nature Genetics, 2019.

[https://github.com/parklab/SigMA](https://github.com/parklab/SigMA)

MosaicForecast(https://github.com/parklab/MosaicForecast)For identification of somatic mutation from bulk whole-genome sequencing data. See Dou et al, in press.

[https://github.com/parklab/MosaicForecast](https://github.com/parklab/MosaicForecast)

LiRA(https://github.com/parklab/LiRA)For identification of somatic mutation from single cell DNA whole-genome sequencing data using phasing information (calls are made for phasable regions). See Bohrson et al, Nature Genetics, 2019.

[https://github.com/parklab/LiRA](https://github.com/parklab/LiRA)

SCAN-SNV(https://github.com/parklab/scan-snv)Single cell genotyper for whole-genome sequencing data (calls are made for the whole genome). See Luquette et al, Nature Communication, 2019.

[https://github.com/parklab/scan-snv](https://github.com/parklab/scan-snv)

ShatterSeek(https://github.com/parklab/ShatterSeek)For identification of chromothripsis events from whole-genome sequencing data. See Ciriano-Cortes et al, Nature Genetics, in press.

[https://github.com/parklab/ShatterSeek](https://github.com/parklab/ShatterSeek)

Tibanna(https://github.com/4dn-dcic/tibanna)For running genomic pipelines on Amazon Web Services (AWS). It supports CWL/WDL (w/ docker), Snakemake (w/ conda) and custom Docker/shell commands. See Lee et al, Bioinformatics, 2019.

[https://github.com/4dn-dcic/tibanna](https://github.com/4dn-dcic/tibanna)

xTEA(available soon)An improved version of our software package TEA for detecting transposable elements from whole-genome data

HiNT(https://github.com/parklab/HiNT)For identification of copy number and structural variants from Hi-C data.

[https://github.com/parklab/HiNT](https://github.com/parklab/HiNT)

## Whole-genome analysis

Meerkat(http://compbio.med.harvard.edu/Meerkat/)This program identifies structural variations from whole-genome sequencing data using patterns of discordant read clusters

[http://compbio.med.harvard.edu/Meerkat/](http://compbio.med.harvard.edu/Meerkat/)

Tea (Transposable Element Analyzer)(http://compbio.med.harvard.edu/Tea/)This is a tool for detection of retrotransposition events in whole-genome sequencing data.

[http://compbio.med.harvard.edu/Tea/](http://compbio.med.harvard.edu/Tea/)

BIC-seq2 (Bayesian Information Criteria-seq)(http://compbio.med.harvard.edu/BIC-seq)This program (R code) is for identification of copy number variants in whole-genome sequencing data. The original paper was published in 2011 (Xi et al, PNAS, 2011). An updated version, BIC-seq2, was published in 2016 (Xi et al, Nucleic Acids Research, 2016). The main difference in BIC-seq2 is to not require a paired control, i.e., it can be used both for somatic and germline CNV calls.

[http://compbio.med.harvard.edu/BIC-seq](http://compbio.med.harvard.edu/BIC-seq/)

## Epigenetics

Antibody Validation Database(http://compbio.med.harvard.edu/antibodies/)This site contains antibody validation data from ENCODE, modENCODE, and Epigenome Roadmap projects. See Egelhoferet al(2011) for details

[http://compbio.med.harvard.edu/antibodies/](http://compbio.med.harvard.edu/antibodies/)

ChIP-seq analysis (SPP)(http://compbio.med.harvard.edu/Supplements/ChIP-seq)This R package by Peter Kharchenko implements tools for analysis of sequencing data from chromatin immunoprecipitation experiments. It includes normalization of the binding profile, detection of enriched regions, and an estimate of read depth needed to achieve saturation of binding sites. See Kharchenkoet al,Nature Biotechnology(2008) for details.

[http://compbio.med.harvard.edu/Supplements/ChIP-seq](http://compbio.med.harvard.edu/Supplements/ChIP-seq)

Repeat enrichment estimator(http://compbio.med.harvard.edu/repeats/)This tool aims to measure the enrichment of annotated repeat types in ChIP-seq data

[http://compbio.med.harvard.edu/repeats/](http://compbio.med.harvard.edu/repeats/)

Quantized correlation coefficient (QCC)(https://compbio.med.harvard.edu/Supplements/BMCBioinfo10.html)This R package computes a robust measurement of the reproducibility of ChIP-chip data.

[https://compbio.med.harvard.edu/Supplements/BMCBioinfo10.html](http://compbio.hms.harvard.edu/peng-et-al-bmc-bioinformatics-2010)

ChIP-chip normalizationThe R package is availablehere (v1.0.1). Here is theinstruction sheet(part of the package).

[here (v1.0.1)](http://compbio.hms.harvard.edu/files/parklab/files/chipchipnorm_1.0.1.tar.gz)

[instruction sheet](http://compbio.hms.harvard.edu/files/parklab/files/chipchipnorm.pdf)

## RNA-seq data

EMSAR(https://github.com/parklab/emsar)This program quantifies gene and mRNA isoform expression levels from RNA-seq data.

[https://github.com/parklab/emsar](https://github.com/parklab/emsar)

## Data Visualization

Nozzle(https://confluence.broadinstitute.org/display/GDAC/Nozzle)A report generation toolkit for data analysis pipelines.

[https://confluence.broadinstitute.org/display/GDAC/Nozzle](https://confluence.broadinstitute.org/display/GDAC/Nozzle)

Also available at CRAN:http://cran.r-project.org/web/packages/Nozzle.R1/index.html. The source code is available athttps://github.com/parklab/Nozzle.

[http://cran.r-project.org/web/packages/Nozzle.R1/index.html](http://cran.r-project.org/web/packages/Nozzle.R1/index.html)

[https://github.com/parklab/Nozzle](https://github.com/parklab/Nozzle)

StratomeX: Visual Analysis of Large-Scale Heterogeneous Genomics Data for Cancer Subtype(http://compbio.med.harvard.edu/tcga/stratomex/)This is a visual exploration tool for identification and characterization of clusters and correlations in genomics data

[http://compbio.med.harvard.edu/tcga/stratomex/](http://compbio.med.harvard.edu/tcga/stratomex/)

modENCODE chromatin data browser(http://compbio.med.harvard.edu/flychromatin/)This website allows one to explore the enrichment profiles of histone marks and chromosomal proteins in theDrosophilagenome.

[http://compbio.med.harvard.edu/flychromatin/](http://compbio.med.harvard.edu/flychromatin/)

## Others

NGSCheckMate(https://github.com/parklab/NGSCheckMate)Software for validating sample identity in next-generation sequencing studies within and across data types. It works for a variety of data types, including whole-genomes, whole-exomes, RNA-seq, ChIP-seq, and targeted panel.

[https://github.com/parklab/NGSCheckMate](https://github.com/parklab/NGSCheckMate)

CGHweb(http://compbio.med.harvard.edu/CGHweb)This tool provides an interface to apply several popular algorithms to segment a copy-number profile from CGH (comparative genomic hybridization) data. It generates a heatmap panel of the segmented profiles for each method as well as a consensus profile. The clickable heatmap can be moved along the chromosome and zoomed in or out. It also displays the time that each algorithm took and provides numerical values of the segmented profiles for download.

[http://compbio.med.harvard.edu/CGHweb](http://compbio.med.harvard.edu/CGHweb)

nuScore(http://compbio.med.harvard.edu/nuScore)This allows estimation of the affinity of the histone core to DNA and prediction of nucleosome arrangement on a given sequence. The algorithm is based on assessment of the energy cost of imposing the deformations required to wrap DNA around the histone surface

[http://compbio.med.harvard.edu/nuScore](http://compbio.med.harvard.edu/nuScore/)

CrossChipThis tool is for integrating data from multiple generations of Affymetrix GeneChips. Matching probes based on Affymetrix "best-match" is inadequate for most analyses. This tool allows the user to derive a list of similar probes based on the user-specified criteria on probe sequence similarity and the minimum number of probe pairs needed for each probe set.

sigPathway(for finding significant pathways from microarray data)For the R package, please see theSupplementary Material pagefor the Tian et al article (Proc Natl Acad Sci,2005).

[Supplementary Material page](http://compbio.hms.harvard.edu/tian-et-al-proc-natl-acad-sci-usa-2005)