
<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <title>Supplements | Park Lab | Computational Genomics | Harvard</title>
    <link rel="stylesheet" href="/assets/css/app.css">
    <link rel="shortcut icon" type="image/png" href="/img/favicon.svg" />
    <link
        href="https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600;700&family=Montserrat:wght@800&display=swap"
        rel="stylesheet"
    />
    <script defer src="https://use.fontawesome.com/releases/v5.3.1/js/all.js"></script>
    <!-- Begin Jekyll SEO tag v2.8.0 -->
<title>Supplements | PARK LAB</title>
<meta name="generator" content="Jekyll v3.8.7" />
<meta property="og:title" content="Supplements" />
<meta name="author" content="Author" />
<meta property="og:locale" content="en_US" />
<meta name="description" content="Supplements Description" />
<meta property="og:description" content="Supplements Description" />
<link rel="canonical" href="http://example.com/supplements/" />
<meta property="og:url" content="http://example.com/supplements/" />
<meta property="og:site_name" content="PARK LAB" />
<meta property="og:type" content="website" />
<meta name="twitter:card" content="summary" />
<meta property="twitter:title" content="Supplements" />
<meta name="twitter:site" content="@twitter_username" />
<meta name="twitter:creator" content="@Author" />
<script type="application/ld+json">
{"@context":"https://schema.org","@type":"WebPage","author":{"@type":"Person","name":"Author"},"description":"Supplements Description","headline":"Supplements","url":"http://example.com/supplements/"}</script>
<!-- End Jekyll SEO tag -->
<script async src="https://www.googletagmanager.com/gtag/js?id=GoogleAnalytics"></script>
<script>
  window['ga-disable-GoogleAnalytics'] = window.doNotTrack === "1" || navigator.doNotTrack === "1" || navigator.doNotTrack === "yes" || navigator.msDoNotTrack === "1";
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());
  gtag('config', 'GoogleAnalytics');
</script><!-- head scripts --></head>

  <body>
    
<nav class="navbar is-primary" >
    <div class="container">
        <div class="navbar-brand">
            <a href="/" class="navbar-item"><b>
                PARK LAB
            </b></a>
            <a role="button" class="navbar-burger burger" aria-label="menu" aria-expanded="false" data-target="navMenu">
                <span aria-hidden="true"></span>
                <span aria-hidden="true"></span>
                <span aria-hidden="true"></span>
            </a>
        </div>
        <div class="navbar-menu" id="navMenu">
            <div class="navbar-content">
                
                
                    <!--  -->
                    <div class="navbar-item">
                        <a href="/news/" class="navbar-item-link link-underline-animate ">
                            <span class="navbar-item-text">
                                News
                            </span>
                        </a>
                        <div class="disc"></div>
                    </div>
                    
                
                    <!--  -->
                    <div class="navbar-item">
                        <a href="/research/" class="navbar-item-link link-underline-animate ">
                            <span class="navbar-item-text">
                                Research
                            </span>
                        </a>
                        <div class="disc"></div>
                    </div>
                    
                
                    <!--  -->
                    <div class="navbar-item">
                        <a href="/people/" class="navbar-item-link link-underline-animate ">
                            <span class="navbar-item-text">
                                People
                            </span>
                        </a>
                        <div class="disc"></div>
                    </div>
                    
                
                    <!--  -->
                    <div class="navbar-item">
                        <a href="/publications/" class="navbar-item-link link-underline-animate ">
                            <span class="navbar-item-text">
                                Publications
                            </span>
                        </a>
                        <div class="disc"></div>
                    </div>
                    
                
                    <!--  -->
                    <div class="navbar-item">
                        <a href="/supplements/" class="navbar-item-link link-underline-animate is-active ">
                            <span class="navbar-item-text">
                                Supplements
                            </span>
                        </a>
                        <div class="disc"></div>
                    </div>
                    
                
                    <!--  -->
                    <div class="navbar-item">
                        <a href="/software/" class="navbar-item-link link-underline-animate ">
                            <span class="navbar-item-text">
                                Software
                            </span>
                        </a>
                        <div class="disc"></div>
                    </div>
                    
                
                    <!--  -->
                    <div class="navbar-item">
                        <a href="/positions/" class="navbar-item-link link-underline-animate ">
                            <span class="navbar-item-text">
                                Positions
                            </span>
                        </a>
                        <div class="disc"></div>
                    </div>
                    
                
                    <!--  -->
                    <div class="navbar-item">
                        <a href="/contact/" class="navbar-item-link link-underline-animate ">
                            <span class="navbar-item-text">
                                Contact
                            </span>
                        </a>
                        <div class="disc"></div>
                    </div>
                    
                
                
            </div>
        </div>
    </div>
</nav>

    
    


    <section class="section">
        <div class="container">
            
            <div class="columns">
                
                <div class="column is-12">
                    
                    
                    
<div class="page-supplements content">
    <h1 id="supplementary-material-for-published-papers">Supplementary Material for Published Papers</h1>



<div class="supplements-container">
    
    
    <div class="supplement-container">
        <a href="./wang-et-al-genome-biology-2020">Wang et al, HiNT: a computational method for detecting copy number variations and translocations from Hi-C data, Genome Biology, 2020</a>

        
        <br/>
        <b>Abstract</b>: The three-dimensional conformation of a genome can be profiled using Hi-C, a technique that combines chromatin conformation capture with high-throughput sequencing. However, structural variations often yield features that can be mistaken for chromosomal interactions. Here, we describe a computational method HiNT (Hi-C for copy Number variation and Translocation detection), which detects copy number variations and interchromosomal translocations within Hi-C data with breakpoints at single base-pair resolution. We demonstrate that HiNT outperforms existing methods on both simulated and real data. We also show that Hi-C can supplement whole-genome sequencing in structure variant detection by locating breakpoints in repetitive regions.
        

        
    </div>
    
    <div class="supplement-container">
        <a href="https://urldefense.proofpoint.com/v2/url?u=https-3A__www.ncbi.nlm.nih.gov_projects_gap_cgi-2Dbin_study.cgi-3Fstudy-5Fid-3Dphs001485.v1.p1&d=DwMGaQ&c=WO-RGvefibhHBZq3fL85hQ&r=DgT-intmgxu2uQxR80ko984jWjwRJewF-vRbWa2mk8o&m=R5a81a12m9sDaK9p6z_rmypb9xfd_2B8FOw-6ZeIoUQ&s=WyxsjxDBWQefkKwSPTDn1mdZcjkM7AxhKA9nOhb8drs&e=">Lodato et al, Aging and neurodegeneration are associated with increased mutations in single human neurons, Science, 2018.</a>

        

        
        <br/>
        <p>Data are available from dbGaP: https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs001485.v1.p1</p>
        
    </div>
    
    <div class="supplement-container">
        <a href="https://www.encodeproject.org/comparative/">Ho et al, Comparative analysis of metazoan chromatin organization, Nature, 2014</a>

        
        <br/>
        <b>Abstract</b>: Genome function is dynamically regulated in part by chromatin, which consists of the histones, non-histone proteins and RNA molecules that package DNA. Studies in Caenorhabditis elegans (worm) and Drosophila melanogaster (fly) have contributed notably to our understanding of molecular mechanisms of genome function in humans, and have revealed conservation of chromatin components and mechanisms. Nevertheless, the three organisms have markedly different genome sizes, chromosome architecture and gene organization. On human and fly chromosomes, for example, pericentric heterochromatin flanks single centromeres, whereas worm chromosomes have dispersed heterochromatin-like regions enriched in the distal chromosomal ‘arms’, and centromeres distributed along their lengths. To systematically investigate chromatin organization and associated gene regulation across species, we generated and analysed a large collection of genome-wide chromatin data sets from cell lines and developmental stages in worm, fly and human. Here we present over 800 new data sets from our ENCODE and modENCODE consortia, bringing the total to over 1,400. Comparison of combinatorial patterns of histone modifications, nuclear lamina-associated domains, the organization of large-scale topological domains, the chromatin environment at promoters and enhancers, nucleosome positioning, and DNA replication patterns reveals many conserved features of chromatin organization among the three organisms. We also find notable differences in the composition and locations of repressive chromatin. These data sets and analyses provide a rich resource for comparative and species-specific investigations of chromatin composition, organization and function.
        

        
    </div>
    
    <div class="supplement-container">
        <a href="./supplementary-material-apostolou-et-al-2013">Apostolou et al, Genome-wide Chromatin Interactions of the Nanog Locus in Pluripotency, Differentiation, and Reprogramming, Cell Stem Cell, 2013</a>

        
        <br/>
        <b>Abstract</b>: The chromatin state of pluripotency genes has been studied extensively in embryonic stem cells (ESCs) and differentiated cells, but their potential interactions with other parts of the genome remain largely unexplored. Here, we identified a genome-wide, pluripotency-specific interaction network around the Nanog promoter by adapting circular chromosome conformation capture sequencing. This network was rearranged during differentiation and restored in induced pluripotent stem cells. A large fraction of Nanog-interacting loci were bound by Mediator or cohesin in pluripotent cells. Depletion of these proteins from ESCs resulted in a disruption of contacts and the acquisition of a differentiation-specific interaction pattern prior to obvious transcriptional and phenotypic changes. Similarly, the establishment of Nanog interactions during reprogramming often preceded transcriptional upregulation of associated genes, suggesting a causative link. Our results document a complex, pluripotency-specific chromatin “interactome” for Nanog and suggest a functional role for long-range genomic interactions in the maintenance and induction of pluripotency.
        

        
    </div>
    
    <div class="supplement-container">
        <a href="./tolstorukov-et-al-nature-structural-and-molecular-biology-2011">Tolstorukov et al, Impact of chromatin structure on sequence variability in the human genome, Nature Structural and Molecular Biology, 2011</a>

        
        <br/>
        <b>Abstract</b>: DNA sequence variations in individual genomes give rise to different phenotypes within the same species. One mechanism in this process is the alteration of chromatin structure due to sequence variation that impacts gene regulation. We composed a high-confidence collection of human SNPs and indels based on analysis of publicly available sequencing data and investigated whether the DNA loci associated with stable nucleosome positions are protected against mutations. We addressed how the sequence variation is reflected in the occupancy profiles of nucleosomes bearing different epigenetic modifications on genome scale. We find that indels are depleted around nucleosome positions of all considered types, while SNPs are enriched around the positions of bulk nucleosomes but depleted around the positions of epigenetically modified nucleosomes. These findings indicate an increased level of conservation for the sequences associated with epigenetically modified nucleosomes, highlighting complex organization of the human chromatin.
        

        
    </div>
    
    <div class="supplement-container">
        <a href="./larschan-et-al-nature-2011">Larschan et al, X chromosome dosage compensation via enhanced transcriptional elongation in Drosophila Nature, 2011</a>

        
        <br/>
        <b>Abstract</b>: The evolution of sex chromosomes has resulted in numerous species in which females inherit two X chromosomes but males have a single X, thus requiring dosage compensation. MSL (Male-specific lethal) complex increases transcription on the single X chromosome of Drosophila males to equalize expression of X-linked genes between the sexes. The biochemical mechanisms used for dosage compensation must function over a wide dynamic range of transcription levels and differential expression patterns. It has been proposed that the MSL complex regulates transcriptional elongation to control dosage compensation, a model subsequently supported by mapping of the MSL complex and MSL-dependent histone 4 lysine 16 acetylation to the bodies of X-linked genes in males, with a bias towards 3′ ends. However, experimental analysis of MSL function at the mechanistic level has been challenging owing to the small magnitude of the chromosome-wide effect and the lack of an in vitro system for biochemical analysis. Here we use global run-on sequencing (GRO-seq) to examine the specific effect of the MSL complex on RNA Polymerase II (RNAP II) on a genome-wide level. Results indicate that the MSL complex enhances transcription by facilitating the progression of RNAP II across the bodies of active X-linked genes. Improving transcriptional output downstream of typical gene-specific controls may explain how dosage compensation can be imposed on the diverse set of genes along an entire chromosome.
        

        
    </div>
    
    <div class="supplement-container">
        <a href="./kim-et-al-cancer-research-2011">Kim et al, A developmental taxonomy of glioblastoma defined and maintained by microRNAs, Cancer Research, 2011</a>

        
        <br/>
        <b>Abstract</b>: mRNA expression profiling has suggested the existence of multiple glioblastoma subclasses, but their number and characteristics vary among studies and the etiology underlying their development is unclear. In this study, we analyzed 261 microRNA expression profiles from The Cancer Genome Atlas (TCGA), identifying five clinically and genetically distinct subclasses of glioblastoma that each related to a different neural precursor cell type. These microRNA-based glioblastoma subclasses displayed microRNA and mRNA expression signatures resembling those of radial glia, oligoneuronal precursors, neuronal precursors, neuroepithelial/neural crest precursors, or astrocyte precursors. Each subclass was determined to be genetically distinct, based on the significant differences they displayed in terms of patient race, age, treatment response, and survival. We also identified several microRNAs as potent regulators of subclass-specific gene expression networks in glioblastoma. Foremost among these is miR-9, which suppresses mesenchymal differentiation in glioblastoma by downregulating expression of JAK kinases and inhibiting activation of STAT3. Our findings suggest that microRNAs are important determinants of glioblastoma subclasses through their ability to regulate developmental growth and differentiation programs in several transformed neural precursor cell types. Taken together, our results define developmental microRNA expression signatures that both characterize and contribute to the phenotypic diversity of glioblastoma subclasses, thereby providing an expanded framework for understanding the pathogenesis of glioblastoma in a human neurodevelopmental context.
        

        
    </div>
    
    <div class="supplement-container">
        <a href="./kharchenko-et-al-nature-2011">Kharchenko et al, Comprehensive analysis of the chromatin landscape in Drosophila melanogaster, Nature, 2011</a>

        
        <br/>
        <b>Abstract</b>: Chromatin is composed of DNA and a variety of modified histones and non-histone proteins, which have an impact on cell differentiation, gene regulation and other key cellular processes. Here we present a genome-wide chromatin landscape for Drosophila melanogaster based on eighteen histone modifications, summarized by nine prevalent combinatorial patterns. Integrative analysis with other data (non-histone chromatin proteins, DNase I hypersensitivity, GRO-Seq reads produced by engaged polymerase, short/long RNA products) reveals discrete characteristics of chromosomes, genes, regulatory elements and other functional domains. We find that active genes display distinct chromatin signatures that are correlated with disparate gene lengths, exon patterns, regulatory functions and genomic contexts. We also demonstrate a diversity of signatures among Polycomb targets that include a subset with paused polymerase. This systematic profiling and integrative analysis of chromatin signatures provides insights into how genomic elements are regulated, and will serve as a resource for future experimental investigations of genome structure and function.
        

        
    </div>
    
    <div class="supplement-container">
        <a href="./peng-et-al-bmc-bioinformatics-2010">Peng et al, Quantized correlation coefficient for measuring reproducibility of ChIP-chip data, BMC Bioinformatics, 11:399, 2010</a>

        
        <br/>
        <b>Abstract</b>: Chromatin immunoprecipitation followed by microarray hybridization (ChIP-chip) is used to study protein-DNA interactions and histone modifications on a genome-scale. To ensure data quality, these experiments are usually performed in replicates, and a correlation coefficient between replicates is used often to assess reproducibility. However, the correlation coefficient can be misleading because it is affected not only by the reproducibility of the signal but also by the amount of binding signal present in the data. We develop the Quantized correlation coefficient (QCC) that is much less dependent on the amount of signal. This involves discretization of data into set of quantiles (quantization), a merging procedure to group the background probes, and recalculation of the Pearson correlation coefficient. This procedure reduces the influence of the background noise on the statistic, which then properly focuses more on the reproducibility of the signal. The performance of this procedure is tested in both simulated and real ChIP-chip data. For replicates with different levels of enrichment over background and coverage, we find that QCC reflects reproducibility more accurately and is more robust than the standard Pearson or Spearman correlation coefficients. The quantization and the merging procedure can also suggest a proper quantile threshold for separating signal from background for further analysis.
        

        
    </div>
    
    <div class="supplement-container">
        <a href="./kim-et-al-bmc-bioinformatics-2010">Kim et al, rSW-seq: Algorithm for detection of copy number alterations in deep sequencing data, BMC Bioinformatics, 11:432, 2010</a>

        
        <br/>
        <b>Abstract</b>: We develop a method for identification of copy number alterations in a tumor genome compared to its matched control, based on application of Smith-Waterman algorithm to single-end sequencing data. In a performance test with simulated data, our algorithm shows >90% sensitivity and >90% precision in detecting a single copy number change that contains approximately 500 reads for the normal sample. With 100-bp reads, this corresponds to a ~50 kb region for 1X genome coverage of the human genome. We further refine the algorithm to develop rSW-seq, (recursive Smith-Waterman-seq) to identify alterations in a complex configuration, which are commonly observed in the human cancer genome. To validate our approach, we compare our algorithm with an existing algorithm using simulated and publicly available datasets. We also compare the sequencing-based profiles to microarray-based results.
        

        
    </div>
    
    <div class="supplement-container">
        <a href="./park-et-al-computational-statistics-and-data-analysis-2009">Park et al, A Permutation Test for determining significance of clusters with applications to spatial and gene expression data, Computational Statistics and Data Analysis, 2009</a>

        
        <br/>
        <b>Abstract</b>: Hierarchical clustering is a common procedure for identifying structure in a dataset, and this is frequently used for organizing genomic data. Although more advanced clustering algorithms are available, the simplicity and visual appeal of hierarchical clustering have made it ubiquitous in gene expression data analysis. Hence, even minor improvements in this framework would have significant impact. There is currently no simple and systematic way of assessing and displaying the significance of various clusters in a resulting dendrogram without making certain distributional assumptions or ignoring gene-specific variances. In this work, we introduce a permutation test based on comparing the within-cluster structure of the observed data with those of sample datasets obtained by permuting the cluster membership. We carry out this test at each node of the dendrogram using a statistic derived from the singular value decomposition of variance matrices. The p-values thus obtained provide insight into the significance of each cluster division. Given these values, one can also modify the dendrogram by combining non-significant branches. By adjusting the cut-off level of significance for branches, one can produce dendrograms with a desired level of detail for ease of interpretation. We demonstrate the usefulness of this approach by applying it to illustrative datasets.
        

        
    </div>
    
    <div class="supplement-container">
        <a href="./gelbart-et-al-nature-structural-and-molecular-biology-2009">Gelbart et al, Drosophila MSL complex globally acetylates H4K16 on the male X chromosome for dosage compensation, Nature Structural and Molecular Biology, 2009</a>

        
        <br/>
        <b>Abstract</b>: The Drosophila melanogaster male-specific lethal (MSL) complex binds the single male X chromosome to upregulate gene expression to equal that from the two female X chromosomes. However, it has been puzzling that approximately 25% of transcribed genes on the X chromosome do not stably recruit MSL complex. Here we find that almost all active genes on the X chromosome are associated with robust H4 Lys16 acetylation (H4K16ac), the histone modification catalyzed by the MSL complex. The distribution of H4K16ac is much broader than that of the MSL complex, and our results favor the idea that chromosome-wide H4K16ac reflects transient association of the MSL complex, occurring through spreading or chromosomal looping. Our results parallel those of localized Polycomb repressive complex and its more broadly distributed chromatin mark, trimethylated histone H3 Lys27 (H3K27me3), suggesting a common principle for the establishment of active and silenced chromatin domains.
        

        
    </div>
    
    <div class="supplement-container">
        <a href="./dreyfuss-et-al-molecular-cancer-2009">Dreyfuss et al, Meta-analysis of Glioblastoma multiforme versus Anaplastic astrocytoma identifies robust gene markers, Molecular Cancer, 8:71, 2009</a>

        
        <br/>
        <b>Abstract</b>: BackgroundAnaplastic astrocytoma (AA) and its more aggressive counterpart, glioblastoma multiforme (GBM), are the most common intrinsic brain tumors in adults and are almost universally fatal. A deeper understanding of the molecular relationship of these tumor types is necessary to derive insights into the diagnosis, prognosis, and treatment of gliomas. Although genomewide profiling of expression levels with microarrays can be used to identify differentially expressed genes between these tumor types, comparative studies so far have resulted in gene lists that show little overlap.ResultsTo achieve a more accurate and stable list of the differentially expressed genes and pathways between primary GBM and AA, we performed a meta-analysis using publicly available genome-scale mRNA data sets. There were four data sets with sufficiently large sample sizes of both GBMs and AAs, all of which coincidentally used human U133 platforms from Affymetrix, allowing for easier and more precise integration of data. After scoring genes and pathways within each data set, we combined the statistics across studies using the nonparametric rank sum method to identify the features that differentiate GBMs and AAs. We found >900 statistically significant probe sets after correction for multiple testing from the >22,000 tested. We also used the rank sum approach to select >20 significant Biocarta pathways after correction for multiple testing out of >175 pathways examined. The most significant pathway was the hypoxia-inducible factor (HIF) pathway. Our analysis suggests that many of the most statistically significant genes work together in a HIF1A/VEGF-regulated network to increase angiogenesis and invasion in GBM when compared to AA.ConclusionWe have performed a meta-analysis of genome-scale mRNA expression data for 289 human malignant gliomas and have identified a list of >900 probe sets and >20 pathways that are significantly different between GBM and AA. These feature lists could be utilized to aid in diagnosis, prognosis, and grade reduction of high-grade gliomas and to identify genes that were not previously suspected of playing an important role in glioma biology. More generally, this approach suggests that combined analysis of existing data sets can reveal new insights and that the large amount of publicly available cancer data sets should be further utilized in a similar manner.
        

        
    </div>
    
    <div class="supplement-container">
        <a href="./sural-et-al-nature-structural-and-molecular-biology-2008">Sural et al, The MSL3 chromodomain directs a key targeting step for dosage compensation of the Drosophila melanogaster X chromosome,Nature Structural and Molecular Biology, 2008</a>

        
        <br/>
        <b>Abstract</b>: The male-specific lethal (MSL) complex upregulates the single male X chromosome to achieve dosage compensation in Drosophila melanogaster. We have proposed that MSL recognition of specific entry sites on the X is followed by local targeting of active genes marked by histone H3 trimethylation (H3K36me3). Here we analyze the role of the MSL3 chromodomain in the second targeting step. Using ChIP-chip analysis, we find that MSL3 chromodomain mutants retain binding to chromatin entry sites but show a clear disruption in the full pattern of MSL targeting in vivo, consistent with a loss of spreading. Furthermore, when compared to wild type, chromodomain mutants lack preferential affinity for nucleosomes containing H3K36me3 in vitro. Our results support a model in which activating complexes, similarly to their silencing counterparts, use the nucleosomal binding specificity of their respective chromodomains to spread from initiation sites to flanking chromatin.
        

        
    </div>
    
    <div class="supplement-container">
        <a href="./orford-et-al-develpmental-cell-2008">Orford et al, Differential H3K4 methylation identifies developmentally poised hematopoietic genes, Developmental Cell, 2008</a>

        
        <br/>
        <b>Abstract</b>: Throughout development, cell fate decisions are converted into epigenetic information that determines cellular identity. Covalent histone modifications are heritable epigenetic marks and are hypothesized to play a central role in this process. In this report, we assess the concordance of histone H3 lysine 4 dimethylation (H3K4me2) and trimethylation (H3K4me3) on a genome-wide scale in erythroid development by analyzing pluripotent, multipotent, and unipotent cell types. Although H3K4me2 and H3K4me3 are concordant at most genes, multipotential hematopoietic cells have a subset of genes that are differentially methylated (H3K4me2+/me3-). These genes are transcriptionally silent, highly enriched in lineage-specific hematopoietic genes, and uniquely susceptible to differentiation-induced H3K4 demethylation. Self-renewing embryonic stem cells, which restrict H3K4 methylation to genes that contain CpG islands (CGIs), lack H3K4me2+/me3- genes. These data reveal distinct epigenetic regulation of CGI and non-CGI genes during development and indicate an interactive relationship between DNA sequence and differential H3K4 methylation in lineage-specific differentiation.
        

        
    </div>
    
    <div class="supplement-container">
        <a href="./alekseyenko-et-al-cell-2008">Alekseyenko et al, A sequence motif within chromatin entry sites directs MSL establishment on the Drosophila X chromosome, Cell, 2008</a>

        
        <br/>
        <b>Abstract</b>: The Drosophila MSL complex associates with active genes specifically on the male X chromosome to acetylate histone H4 at lysine 16 and increase expression approximately 2-fold. To date, no DNA sequence has been discovered to explain the specificity of MSL binding. We hypothesized that sequence-specific targeting occurs at "chromatin entry sites," but the majority of sites are sequence independent. Here we characterize 150 potential entry sites by ChIP-chip and ChIP-seq and discover a GA-rich MSL recognition element (MRE). The motif is only slightly enriched on the X chromosome ( approximately 2-fold), but this is doubled when considering its preferential location within or 3' to active genes (>4-fold enrichment). When inserted on an autosome, a newly identified site can direct local MSL spreading to flanking active genes. These results provide strong evidence for both sequence-dependent and -independent steps in MSL targeting of dosage compensation to the male X chromosome.
        

        
    </div>
    
    <div class="supplement-container">
        <a href="./peng-et-al-bmc-bioinformatics-2007">Peng et al, Normalization and experimental design for ChIP-chip data, BMC Bioinformatics, 2007</a>

        
        <br/>
        <b>Abstract</b>: BACKGROUND Chromatin immunoprecipitation on tiling arrays (ChIP-chip) has been widely used to investigate the DNA binding sites for a variety of proteins on a genome-wide scale. However, several issues in the processing and analysis of ChIP-chip data have not been resolved fully, including the effect of background (mock control) subtraction and normalization within and across arrays.
        

        
    </div>
    
    <div class="supplement-container">
        <a href="./larschan-et-al-molecular-cell-2007">Larschan et al, MSL complex is attracted to genes marked by H3K36 trimethylation using a sequence-independent mechanism, Molecular Cell, 2007</a>

        
        <br/>
        <b>Abstract</b>: Dosage compensation equalizes the levels of transcripts encoded on the X chromosome between XY males and XX females. In Drosophila, dosage compensation requires the MSL (Male Specific Lethal) complex, which associates with actively transcribed genes on the single male X chromosome to upregulate transcription approximately two-fold. MSL complex targets genes, with increased binding toward 3’ ends. To search for chromatin modifications associated with MSL binding, we mapped H3K36 trimethylation (H3K36me3) in Drosophila SL2 cells, and found that it marks transcribed genes with a 3’ bias, as in yeast and humans. On the male X chromosome, or when MSL complex is ectopically localized to an autosome, H3K36me3 is a strong predictor of MSL binding. We isolated mutants lacking Set2, the H3K36me3 methyltransferase, and found that Set2 is an essential gene in both sexes of Drosophila. In set2 mutant males, MSL complex can still associate with high affinity sites, which are proposed to mark the X chromosome through DNA sequence elements. Yet, MSL complex exhibits reduced binding to target genes, suggesting a specific role for H3K36me3 in MSL targeting. In addition, we found that recombinant MSL3 protein preferentially binds nucleosomes marked by H3K36me3 in vitro. Our results support a model in which the MSL complex recognizes many of its targets through general features of transcribed genes.
        

        
    </div>
    
    <div class="supplement-container">
        <a href="./alekseyenko-et-al-genes-development-2006">Alekseyenko et al, High-resolution ChIP-chip analysis reveals that the Drosophila MSL complex selectively identifies active genes on the male X chromosome, Genes & Development, 2006</a>

        
        <br/>
        <b>Abstract</b>: X-chromosome dosage compensation in Drosophila requires the male-specific lethal (MSL) complex, which up-regulates gene expression from the single male X chromosome. Here, we define X-chromosome-specific MSL binding at high resolution in two male cell lines and in late-stage embryos. We find that the MSL complex is highly enriched over most expressed genes, with binding biased toward the 3 end of transcription units. The binding patterns are largely similar in the distinct cell types, with ~600 genes clearly bound in all three cases. Genes identified as clearly bound in one cell type and not in another indicate that attraction of MSL complex correlates with expression state. Thus, sequence alone is not sufficient to explain MSL targeting. We propose that the MSL complex recognizes most X-linked genes, but only in the context of chromatin factors or modifications indicative of active transcription. Distinguishing expressed genes from the bulk of the genome is likely to be an important function common to many chromatin organizing and modifying activities.
        

        
    </div>
    
    <div class="supplement-container">
        <a href="./tian-et-al-proc-natl-acad-sci-usa-2005">Tian et al, Discovering statistically significant pathways in expression profiling studies, Proc Natl Acad Sci USA, 2005</a>

        
        <br/>
        <b>Abstract</b>: Accurate and rapid identification of perturbed pathways through the analysis of genome-wide expression profiles facilitates the generation of biological hypotheses. We propose a statistical framework for determining whether a specified group of genes for a pathway has a coordinated association with a phenotype of interest. Several issues on proper hypothesis-testing procedures are clarified. In particular, it is shown that the differences in the correlation structure of each set of genes can lead to a biased comparison among gene sets unless a normalization procedure is applied. We propose statistical tests for two important but different aspects of association for each group of genes. This approach has more statistical power than currently available methods and can result in the discovery of statistically significant pathways that are not detected by other methods. This method is applied to data sets involving diabetes, inflammatory myopathies, and Alzheimer's disease, using gene sets we compiled from various public databases. In the case of inflammatory myopathies, we have correctly identified the known cytotoxic T lymphocyte-mediated autoimmunity in inclusion body myositis. Furthermore, we predicted the presence of dendritic cells in inclusion body myositis and of an IFN-alpha/beta response in dermatomyositis, neither of which was previously described. These predictions have been subsequently corroborated by immunohistochemistry.
        

        
    </div>
    
    <div class="supplement-container">
        <a href="./lai-et-al-bioinformatics-2005">Lai et al, Comparative analysis of algorithms for identifying amplifications and deletions in array CGH data, Bioinformatics, 2005</a>

        
        <br/>
        <b>Abstract</b>: MOTIVATION: Array Comparative Genomic Hybridization (CGH) can reveal chromosomal aberrations in the genomic DNA. These amplifications and deletions at the DNA level are important in the pathogenesis of cancer and other diseases. While a large number of approaches have been proposed for analyzing the large array CGH datasets, the relative merits of these methods in practice are not clear. RESULTS: We compare 11 different algorithms for analyzing array CGH data. These include both segment detection methods and smoothing methods, based on diverse techniques such as mixture models, Hidden Markov Models, maximum likelihood, regression, wavelets and genetic algorithms. We compute the Receiver Operating Characteristic (ROC) curves using simulated data to quantify sensitivity and specificity for various levels of signal-to-noise ratio and different sizes of abnormalities. We also characterize their performance on chromosomal regions of interest in a real dataset obtained from patients with Glioblastoma Multiforme. While comparisons of this type are difficult due to possibly sub-optimal choice of parameters in the methods, they nevertheless reveal general characteristics that are helpful to the biological investigator.
        

        
    </div>
    
    <div class="supplement-container">
        <a href="./hamada-et-al-genes-development-2005">Hamada et al, Global regulation of X chromosomal genes by the MSL complex in Drosophila melanogaster, Genes & Development, 2005</a>

        
        <br/>
        <b>Abstract</b>: A long-standing model postulates that X-chromosome dosage compensation in Drosophila occurs by twofold up-regulation of the single male X, but previous data cannot exclude an alternative model, in which male autosomes are down-regulated to balance gene expression. To distinguish between the two models, we used RNA interference to deplete Male-Specific Lethal (MSL) complexes from male-like tissue culture cells. We found that expression of many genes from the X chromosome decreased, while expression from the autosomes was largely unchanged. We conclude that the primary role of the MSL complex is to up-regulate the male X chromosome.
        

        
    </div>
    
</div>
</div>
                </div>
                
            </div>
        </div>
    </section>
    <footer class="footer">
    <span class="footer-text"><b>Park Lab</b>, Department of Biomedical Informatics, Harvard Medical School, 10 Shattuck St, 3rd Floor, Boston, MA 02115</span>
</footer>

    <script src="/assets/js/app.js" type="text/javascript"></script>
  </body>
</html>

